spec/models/avatar_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'carrierwave/test/matchers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.